rules/prevent-abbreviations.js

```diff
@@ -366,8 +366,17 @@ const shouldFix = variable => {
 
 const isShorthandPropertyIdentifier = identifier => {
 	return identifier.parent.type === 'Property' &&
-		identifier.parent.key === identifier &&
-		identifier.parent.shorthand;
+		identifier.parent.shorthand &&
+		(
+			identifier.parent.key === identifier ||
+			// In `babel-eslint` parent.key is not reference of identifier
```

Comment:
Can you open an issue about this on `babel-eslint`? The point of that package is to normalize the AST.

Reply:
you are right, we might not do this check, I've found other cases also broken, let me have a look at their repo first

Reply:
I’m ok with working around it, but we should always also try to get it fixed upstream.

Reply:
Add a code comment with a link to the Babel-eslint issue.